sopel/irc/abstract_backends.py

```diff
@@ -23,6 +23,10 @@ class AbstractIRCBackend(abc.ABC):
     Some methods of this class **MUST** be overridden by a subclass, or the
     backend implementation will not function correctly.
     """
+
+    utf8only: bool = False
+    """Whether we are operating in utf8-only mode."""
+
```

Comment:
Blank line right after docstring is _nyet_ (need to figure out how to make flake8 flag that).

But also, can this be initialized with the other instance values inside `__init__`?